docs/getting_started.md

```diff
@@ -20,6 +26,12 @@ Docker image.
 
 Ubuntu on Windows WSL 2.0 seems to work for most things, but you may encounter errors with more complex operations. For instance, compiling Linux is known not to work in this environment. This is considered an experimental build.
 
+If you are using a newer version of Ubuntu (e.g., 20.04 LTS), you will want to update `setuptools`
```

Comment:
Is there a corresponding PR to add this in SDK README?

Reply:
Not sure. I'm removing this because I think the bsg_sv2v bump actually fixes the issue that was seen related to bsg_sv2v, and this wasn't an SDK thing.